app/src/main/java/net/aiscope/gdd_app/dagger/MicroscopeQualityModule.kt

```diff
@@ -0,0 +1,29 @@
+package net.aiscope.gdd_app.dagger
+
+import android.app.Activity
+import dagger.Binds
+import dagger.Module
+import dagger.Provides
+import net.aiscope.gdd_app.ui.microscope_quality.MicroscopeQualityActivity
+import net.aiscope.gdd_app.ui.microscope_quality.MicroscopeQualityMapper
+import net.aiscope.gdd_app.ui.microscope_quality.MicroscopeQualityView
+
+@Module
+abstract class MicroscopeQualityModule {
+
+    @Binds
+    @PerActivity
+    internal abstract fun provideView(view: MicroscopeQualityActivity): MicroscopeQualityView
+
+    @Binds
+    @PerActivity
+    internal abstract fun activity(activity: MicroscopeQualityActivity): Activity
+
+    @Module
```

Comment:
These `@module` and `@jvmstatic` annotations have recently been removed for other modules. See https://github.com/theaiscope/GDD-app/commit/714614af91441f36e57b73f5e1c96646d0e5bb3f

I'm assuming we also won't need them here?

Reply:
Great catch! Thanks!